index.bs

```diff
@@ -580,8 +648,35 @@ with parameters |session| and |capabilities| is:
 
 </div>
 
+Common Data Types {#data_types}
+===============================
+
+This section defines data types which are common to many modules.
+
+## Browsing Context ## {#data_types-browsing_context}
```

Comment:
```suggestion
## Browsing Context ## {#data-types-browsing-context}
```

I guess this case is why you went with underscores. But generated IDs for similarly hierarchical concepts don't have underscores.